lib/transports/rest_v2.js

```diff
@@ -336,14 +338,15 @@ class RestV2 {
   }
 
   /**
-   * @param {number} order id - unique id of order
+   * @param {Object} options
+   * @param {number} [options.id] - unique id of order
+   * @param {string} [options.clientOid] - user specific order id in RFC 4122 format. only persist for 24 hours, required when order id is not given
    *
    * @return {Promise}
    *
    * @see https://max.maicoin.com/documents/api_list#!/private/getApiV2TradesMyOfOrder
    */
-  tradesOfOrder (orderId) {
-    const options = { id: orderId }
+  tradesOfOrder (options) {
```

Comment:
This is a breaking change, we should bump major version.

Reply:
yes, please see commit message, when I publish, standard version will bump to v2.0.0